tests/pytests/unit/modules/test_baredoc.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+import salt.modules.baredoc as baredoc
+from tests.support.paths import SALT_CODE_DIR
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {
+        baredoc: {
+            "__opts__": {"extension_modules": SALT_CODE_DIR},
+            "__grains__": {"saltpath": SALT_CODE_DIR},
```

Comment:
If we remove these dunders the tests fail?

Nonetheless, looks good.
Just trying to see if it's really required.